tests/bluetooth/df/connectionless_cte_tx/src/common.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Nordic Semiconductor ASA
+ * Copyright (c) 2020 Nordic Semiconductor ASA
```

Comment:
Why was this changed to 2020? Should it perhaps be 2020-2021?

Reply:
Corrected